solidity/hardhat.config.ts

```diff
@@ -99,6 +99,10 @@ const config: HardhatUserConfig = {
       // deploy stub contracts in tests.
       allowUnlimitedContractSize: process.env.TEST_USE_STUBS_TBTC === "true",
     },
+    localhost: {
```

Comment:
Let's name this network `system_tests` to be explicit it is about system tests and not, for example, local developer's box. Note that this is the approach from #471.

Reply:
Already changed by merging `main`.